docs/PumpChecklist.md

```diff
@@ -0,0 +1,213 @@
+## Checklist for Insulin Pump Implementation
+
+(Key:
+
+ - `[x]` available in data protocol/documented in spec and implemented
+ - `[-]` available in data protocol/documented in spec but *not* yet implemented
+ - `[?]` unknown whether available in data protocol/documented in spec; *not* yet implemented
+ - `*[ ]` TODO: needs implementation!
+ - `[ ]` unavailable in data protocol and/or not documented in spec and not yet implemented)
+
+### Required if Present
+
+#### Basals
+
+  - `[ ]` scheduled basal
+    - `[ ]` basal rate intervals with a start time, duration, and rate delivered
+    - `[ ]` name of basal schedule on each scheduled basal rate interval
+  - `[ ]` manual temp basal
+    - `[ ]` basal rate intervals with a start time, duration, and rate delivered
+    - `[ ]` object representing suppressed scheduled basal *for each segment of the basal schedule that the temp basal intersects*
+  - `[ ]` percentage temp basal
+    - `[ ]` basal rate intervals with a start time, duration, percent
+        - `[ ]` rate provided directly OR
+        - `[ ]` rate computed from percent x suppressed.rate
```

Comment:
It doesn't happen very often here, so I'm just using " OR" at the end of a line to indicate when a subset of checklist items are exclusive. If that's not clear enough for you, format-wise, @gniezen LMK if you have another idea for representing suchlike.